### PR TITLE
Add namespace: {{ .Release.Namespace }} to all namespace scoped resources

### PR DESCRIPTION
**Pull request by floriankoch**

### CODE REVIEW

**Comment by flavio** [comment]
@kubewarden/kubewarden-developers seems easy and reasonable. Something we might pull of just in time for rc3?

File: charts/kubewarden-controller/templates/webhooks.yaml [hunk 1]
```diff
@@ -5,6 +5,7 @@ metadata:
   annotations:
     cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/{{ include "kubewarden-controller.fullname" . }}-serving-cert
   name: kubewarden-controller-mutating-webhook-configuration
+  namespace: {{ .Release.Namespace }}
```
Comment (raulcabello) [inline charts/kubewarden-controller/templates/webhooks.yaml:8]: we don't need this here since `MutatingWebhookConfiguration` is not a namespaced resource.

**Review by raulcabello — CHANGES_REQUESTED** [review]
Thanks @floriankoch ! Please have a look at the comment I left

**Comment by floriankoch** [comment]
@raulcabello   fixed

**Review by raulcabello — APPROVED** [review]
LGTM, thanks!

**Review by jvanz — APPROVED** [review]

**Pull request merged**